### PR TITLE
Service Browser extended

### PR DESCRIPTION
**Pull request by odeimaiz**
Services list in the Service Browser accepts now 2 more filters.
- Type
- Category

The resulting list is a combination of all active filters:
image

### CODE REVIEW

File: services/web/client/source/class/qxapp/component/workbench/servicesCatalogue/ServicesCatalogue.js [hunk 1]
```diff
@@ -141,8 +141,11 @@ qx.Class.define("qxapp.component.workbench.servicesCatalogue.ServicesCatalogue",
       }, this);
 
       // create the filter
-      let filterObj = new qxapp.component.workbench.servicesCatalogue.SearchTypeFilter(this.__controller);
+      let filterObj = new qxapp.component.workbench.servicesCatalogue.SearchTypeFilter(this.__controller, ["name"]);
       // set the filter
+      filterObj["bindItem"] = (ctrl, item, id) => {
```
Comment (oetiker) [inline services/web/client/source/class/qxapp/component/workbench/servicesCatalogue/ServicesCatalogue.js]: `filterObj.bindItem = `

File: Makefile [hunk 2]
```diff
@@ -139,6 +139,10 @@ up-swarm-devel:
 	${DOCKER_COMPOSE} -f services/docker-compose.yml -f services/docker-compose.devel.yml -f services/docker-compose.tools.yml config > $(TEMPCOMPOSE).tmp-compose.yml
 	${DOCKER} stack deploy -c $(TEMPCOMPOSE).tmp-compose.yml services
 
+up-webclient-devel: up-swarm-devel remove-intermediate-file
```
Comment (pcrespov) [inline Makefile]: We plan to redo this calls (see #546)
In the meantime, please add ``up-webclient-devel`` in the ``.PHONY`` list (see line 127) and add help documentation above make target (see line 128 as example)

**Comment by pcrespov** [comment]
I like the concept of filtering by labels but as we discussed, I would push this a bit furhter and for something similar to gihub's approach where 
1. the user (or service creator) can define labels, 
1. labels props (e.g. colors) are editable
1. labels are displayed in a separate list or on top of every service 
1. services filter textfield entry can be written by hand or by clicking on specific labels

File: Makefile (hunk 2 above)
Comment (odeimaiz) [reply Makefile]: Done

**Pull request merged**